src/partials/menu.html

```diff
@@ -0,0 +1,24 @@
+<div class="container-header">
+    <div class="header">
+        <a href="#">
+        <svg class="header__icon--film">
+            <use href=./images/sprite.svg#icon-film"></use>
```

Comment:
Nie masz otwarcia cudzysłowu po "<use href=" i dlatego ikonka się nie wyświetla - jak dodasz otwarcie to svg wyświetla się na stronie :)